org-formation/600-access/MetadataDocuments/iatlas-admin.xml

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8"?><md:EntityDescriptor xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata" entityID="JumpCloud"><md:IDPSSODescriptor WantAuthnRequestsSigned="false" protocolSupportEnumeration="urn:oasis:names:tc:SAML:2.0:protocol"><md:KeyDescriptor use="signing"><ds:KeyInfo xmlns:ds="http://www.w3.org/2000/09/xmldsig#"><ds:X509Data><ds:X509Certificate>MIIFjTCCA3WgAwIBAgIVAPbnL5zajnmaprBczI8hq4b6QM1sMA0GCSqGSIb3DQEBCwUAMH4xCzAJBgNVBAYTAlVTMQswCQYDVQQIEwJDTzEQMA4GA1UEBxMHQm91bGRlcjEZMBcGA1UEChMQU2FnZSBCaW9uZXR3b3JrczEZMBcGA1UECxMQSnVtcENsb3VkU0FNTElkUDEaMBgGA1UEAxMRSnVtcENsb3VkU0FNTFVzZXIwHhcNMjEwMTEyMjIyOTQzWhcNMjYwMTEyMjIyOTQzWjB+MQswCQYDVQQGEwJVUzELMAkGA1UECBMCQ08xEDAOBgNVBAcTB0JvdWxkZXIxGTAXBgNVBAoTEFNhZ2UgQmlvbmV0d29ya3MxGTAXBgNVBAsTEEp1bXBDbG91ZFNBTUxJZFAxGjAYBgNVBAMTEUp1bXBDbG91ZFNBTUxVc2VyMIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAv++uvL1Ayfr9Hk/O3voXKyL4ftonKdge4Wckiq0TPva0/MZrWt/HCSNehhn9N/3tcngVPJ3XqtkC8UrKkEjdcRrMu5bZWT8UJ+L4U9uj2bmQg4WbJ39lYKpCZKqkGwClnZ+cOrZwlfzf/CRadDGNTsyrHjSsTBP4JM1ngEnxJNN2Zu9Mxo9zv3pKZs8Ol1NHQ/yJ1rbp4bZRLooaKzzyX8sbdZK86b2uSZXBgsBVm3zTBGtH/Z2v0nml2+WZA48FEgUjW+vRhHGCCzDDR6R7LO2TbY23fCER0aKaVvCBfNP5ywu6WWSt3CVL+b4f8Uf4CMKJv4Sms1VPFqZK5VvyQvv82onUvidMwx74p9VB5W5ukipDrBKbfOW3b7wvyU0mOdv1Qzda0R+nFLZf/AAvguP/SjK3LBDzvrqpmA62X6jH4aRc41MVV+9PFn6D0EsJDItr/XSDLsp05csFPSH9RAqpzzhBkT3x8tZYZwqF5HCnSZN9inY/AqwSkFopb7CxHqmg/its5F0uPC1F8BgLRvBcwRlKI76jJ6bI23l0fyURtPxcbtScR7HC0McQhQwazSyJ3WDD9EHyvsS/trEsJJh+mky1FEGNzMfgZOwRcdbhuP/sZkzuJKuaG/TW/U8YVI318HGo+OBBw7VHtA2dLfVhJboZI790h/DXPdASWDECAwEAAaMCMAAwDQYJKoZIhvcNAQELBQADggIBAChWxxwhD88fGI8wqLXa80RP6P/IC4rEzgLsCpROOY15HEI7km66B7ubZ3Dp5PBBGB9wJ9UU3/po08iIMMlNjGR3usEq/zX6ZW8U2vP3OtYDuMrrn26VDN9gA5H1zQSdtPGbylRKRI39pJWhcl8w3n+Y75pGpHF/OnFwpR10xCobEbA6MH2u9EDSYKbl+e2Hw1twDyQwanyfez9YyvxAFpwyTKWGhIw5h3igXa/mhmobOSNL74OrvLS2LvIWQBPY4ydqIbD9Ya1ssSEKvk2lyoG2IwkPb5HyNSSZvLsBnjB38T3XdaTnrZl7Dde3aSlYgoega5oXp7aOjAI6l53exj+scy5Zd8rId+cU6RiJGQtTCL+r9Ul0VUiDb9NeUyl6ZqsBIHWJ9DBIG0RLhk5tVx2wr+lV+aEXaRCnNPgPhY8xNuJwpPd5ynODDe4UIZDxteenVMtHj3IFuGSi2JKGiamAha03RjWz3PC8c+qtt8eQ2CQL20F57XElpXMNY1wrhDSoB6cZghro/dneyn3Hhio+esexH3Tb2nMJaPeRq9S7MbaDIccZK3UMAjT5GRzqlWVt9Bz8FlEUZ7BWFn9kBxIr5h3DAi1TByGkQtj1F/1Khjq5BQTlsI5OWbesEfEQlCKmOHcqYAhgDgMcK3krQU4DlZ29BLYacf8+SVjMC2pa</ds:X509Certificate></ds:X509Data></ds:KeyInfo></md:KeyDescriptor><md:NameIDFormat>urn:oasis:names:tc:SAML:2.0:nameid-format:persistent</md:NameIDFormat><md:SingleSignOnService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" Location="https://sso.jumpcloud.com/saml2/testitorgmgnt-admin"/><md:SingleSignOnService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect" Location="https://sso.jumpcloud.com/saml2/testitorgmgnt-admin"/></md:IDPSSODescriptor></md:EntityDescriptor>
```

Comment:
the metadata file is created in JC it can only be created after creating account. will need to update this file. this is only a placeholder.

Reply:
done